adbms/src/main/scala/de/up/hpi/informationsystems/adbms/Dactor.scala

```diff
@@ -13,6 +21,12 @@ abstract class Dactor(name: String) extends Actor with ActorLogging {
   protected val relations: Map[String, Relation]
 
   override def preStart(): Unit = log.info(s"${this.getClass.getSimpleName}($name) started")
-  
+
   override def postStop(): Unit = log.info(s"${this.getClass.getSimpleName}($name) stopped")
+
+  protected def dactorOf(clazz: Class[_], id: Int): ActorRef =
+    Dactor.refFor(context.system, clazz, id)
+
+  protected def dactorSelection(clazz: Class[_], id: Int): ActorSelection =
+    context.system.actorSelection(context.system / Dactor.nameFor(clazz, s"$id"))
```

Comment:
Could you add scaladoc comments for these two methods. As I understand it, `dactorOf` creates a new instance of the class at hand while `dactorSelection` can be used to generate an `actorSelection` for this specific type of `Dactor`'s? What then is the `id` for?